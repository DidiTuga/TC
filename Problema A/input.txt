6
3
1 2 6
3
3 4 6
10
1 _ 2
1 _ 3
2 a 4
2 b 1
3 _ 2
3 a 4
3 a 5
3 b 4
4 a 2
6 a 6
bbaa