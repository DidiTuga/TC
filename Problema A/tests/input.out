NDFA
YES
1 2 1 3 4 2 4
